src/components/header/AccountMenu.tsx

```diff
@@ -52,6 +64,10 @@ export const AccountMenu = () => {
         setOpen(false)
         navigate('/change-pin')
         break
+      case 'fund':
+        setOpen(false)
```

Comment:
We should probably have all these options setOpen(false) right? Should we move this down to after the switch?